test/lib/assert-opts.spec.js

```diff
@@ -0,0 +1,71 @@
+/**
+ * @module test/lib/assert-opts.spec
+ */
```

Comment:
(that being said, it's consistent is seems in the repo, but it doesn't _mean_ anything to me)